akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -467,6 +470,14 @@ object Multipart {
           FastFuture.successful(this)
         override def productPrefix = "FormData.BodyPart.Strict"
       }
+
+      /** Java API */
+      object StrictBuilder {
+        def apply(_name: String, _entity: HttpEntity.Strict,
```

Comment:
I see, then it can be `/** INTERNAL API */ private[akka]`, if not meant for users